pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -0,0 +1,141 @@
+package totalaccountwatcher
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
```

Comment:
Can you sort this imports to group together 
1) Local packages
2) k8s packges
3) other

Reply:
Done